pubspec.yaml

```diff
@@ -74,7 +74,7 @@ dependencies:
   wakelock: ^0.6.2
   flutter_dotenv: ^5.0.2
   flutter_markdown: ^0.6.14
-  pub_release: ^8.0.3
+  # pub_release: ^8.0.3
```

Comment:
Why was this commented out?

Reply:
It was not needed as app dependency.

Reply:
I have removed the commented part.

Reply:
Sorry, I meant remove the dependency entirely

Reply:
Thanks!